ambry-router/src/main/java/com/github/ambry/router/OperationController.java

```diff
@@ -82,7 +82,7 @@
    * @param accountService the {@link AccountService} to use.
    * @throws IOException if the network components could not be created.
    */
-  OperationController(String suffix, String defaultPartitionClass, AccountService accountService,
+  public OperationController(String suffix, String defaultPartitionClass, AccountService accountService,
```

Comment:
Does it need to be public?

Reply:
This needs to be public because otherwise the reflection api used in Utils.obj() method that creates OperationController instance, will not be able to access this constructor.